Sources/AcmeSwift/Models/Account/AccountCredentials.swift

```diff
@@ -2,8 +2,8 @@ import Foundation
 import Crypto
 
 public struct AccountCredentials {
-    private (set) public var contacts: [String] = []
-    private (set) public var key: Crypto.P256.Signing.PrivateKey
+    private(set) public var contacts: [String] = []
+    private(set) public var key: Crypto.P256.Signing.PrivateKey
```

Comment:
The Swift 6 language mode no longer supports spaces here for reasons unclear to me:

<img width="703" alt="image" src="https://github.com/user-attachments/assets/857e2376-5abb-46c1-b192-f10c098502f1">